client/stylesheets/_responsive.scss

```diff
@@ -229,27 +241,55 @@
 		width: 90%;
 		margin: auto;
 	}
+
     .helpAboutUl li:first-child > p {
     	text-align: left !important;
     }
+
+    .faqdiv h3, .safetydiv h3, .reportdiv h3 {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Each selector in a comma sequence should be on its own single line